### ISSUE TITLE
[Accessibility] Word wrap does not work in diff view (F7 and Shift+F7)

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#168746]
CC @isidorn and @meganrogge

Although the recent VSCode update introduced accessibility improvement for word wrap, which used to not work with screen reader mode, I found that the word wrap setting does not take any effect in F7/Shift+F7 (diff view) when  screen reader mode is on.

It would be great if word wrap also works in diff view mode as well.

## System Info

Version: 1.75.0-insider (system setup)
Commit: f076d553210e9ea65e7bccbfa2f7a786c10fcab2
Date: 2022-12-09T09:11:59.691Z
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes

**Comment by isidorn** [comment]
This makes sense to me and I think is technically possible now. It might be that diff view still has hard coded that wrap is disabled.

fyi @alexdima

**Comment by alexdima** [comment]
@isidorn F7/Shift+F7 in a diff editor is a custom rendering for diff chunks, which is triggered from the diff editor, but it is not the diff editor. Adding wrapping support there is a good feature request, but it is unrelated to the recent work we made in the editor.

**Comment by isidorn** [comment]
@alexdima my mistake, forgot that it is a special implementation. Thanks for correcting me.
